website/docs/attached-class.md

```diff
@@ -91,3 +91,69 @@ T.reveal_type(Child.make) # Child
 
 Child.make.say_hi # No error now, as Child.make has the type Child
 ```
+
+## `T.attached_class` as an argument?
+
+At this point, it's natural to think about writing methods that accept arguments
+that take values of type `T.attached_class`; you would get more specific type
+information about the arguments when calling from a sub-class, so why not?
+Consider this example:
```

Comment:
This seems confusing. Maybe just jump to the point?

> Using `T.attached_class` as an argument won't work. To see why, let's look at an example:

Reply:
Good idea. I was a little worried that this introduction would lead people to believe that it was possible, and that if you just skimmed the docs you'd be confused about the errors you'd get.